test/SymbolGraph/ClangImporter/ImportForwardDeclarationsTest.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %empty-directory(%t)
+// RUN: cp -r %S/Inputs/ForwardDeclarations/ForwardDeclarations.framework %t
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -enable-objc-interop -emit-module-path %t/ForwardDeclarations.framework/Modules/ForwardDeclarations.swiftmodule/%target-swiftmodule-name -import-underlying-module -F %t -module-name ForwardDeclarations -disable-objc-attr-requires-foundation-module %s -emit-symbol-graph -emit-symbol-graph-dir %t
```

Comment:
This line shouldn't need the `-emit-symbol-graph` flags since those symbol graphs aren't being tested. You can either remove those flags from this line or copy/paste the FileCheck line between here and the next line.

Reply:
Resolved https://github.com/apple/swift/pull/71947/commits/3d34871ec6c6686ddeba86479af87c2f4da16437